test/sql/insert.sql

```diff
@@ -76,3 +76,17 @@ INSERT INTO one_space_test VALUES
 ('2001-01-01 01:01:01', 1.0, 'device'),
 ('2002-01-01 01:02:01', 1.0, 'device');
 SELECT * FROM one_space_test;
+
+--CTE & EXPLAIN ANALYZE TESTS
+WITH insert_cte as (
+	INSERT INTO one_space_test VALUES
+		('2001-01-01 01:02:01', 1.0, 'device')
+	RETURNING *)
+SELECT * FROM insert_cte;
+
+EXPLAIN (analyze, costs off, timing off, summary off)
+WITH insert_cte as (
+	INSERT INTO one_space_test VALUES
+		('2001-01-01 01:03:01', 1.0, 'device')
+	)
+SELECT 1;
```

Comment:
Since copy seems to use a different code path can we use EXPLAIN with COPY to make sure that works too.

Reply:
No explains on copy.